src/content/roadmap/verkle-trees/index.md

```diff
@@ -0,0 +1,68 @@
+
+## Why do Verkle trees enable smaller witnesses? {#why-do-verkle-trees-enable-smaller-witnesses}
+
+The structure of a Merkle Tree makes witness sizes very large - too large to safely broadcast between peers within a 12 second slot. This is because the witness is a path connecting the data, which is held in trie leaves, to the root hash. To verify the data it is necessary to have not only all the intermediate hashes that connect each leaf to the root, but also all the "sibling" nodes. Each node in the proof has a sibling that it is hashed with to create the next hash up the trie. This is a lot of data. Verkle trees reduce the witness size by shortening the distance between the leaves of the tree and its root and also eliminating the need to provide sibling nodes for verifying the root hash. Even more space efficiency will be gained by using a powerful polynomial commitment scheme instead of the hash-style vector commitment. The polynomial commitment allows the witness to have a fixed size regardless of the number of leaves that it proves.
+
+Under the polynomial commitment scheme, the witnesses have manageable sizes that can easily be transferred on the peer-to-peer network. This allows clients to verify state changes in each block with a minimal amount of data.
+
+<ExpandableCard title="Exactly how much can Verkle trees reduce witness size?">
+
+The witness size varies depending on the number of leaves it includes. Assuming the witness covers 1000 leaves, a witness for a Merkle trie would be about 3.5MB (assuming 7 levels to the trie). A witness for the same data in a Verkle tree (assuming 4 levels to the tree) would be about 150 kB - **about 23x smaller**. This reduction in witness size will allow stateless client witnesses to be acceptably small. Polynomial witnesses are 0.128 -1 kB depending on which specific polynomial commitment is used).
+
+</ExpandableCard>
+
+## What is the structure of a Verkle tree? {#what-is-the-structure-of-a-verkle-tree}
+
+Verkle trees are `(key,value)` pairs where the keys are 32-byte elements composed of a 31-byte _stem_ and a single byte _suffix_. These keys are organized into _extension_ nodes and _inner_ nodes. Extension nodes represent a single stem for 256 children with different suffixes. Inner nodes also have 256 children, but they can be other extension nodes. The main difference between the Verkle tree and the Merkle tree structure is that the Verkle tree is much flatter, meaning there are fewer intermediate nodes linking a leaf to the root, and therefore less data required to generate a proof.
+
+![Illustration of the Verkle tree structure from the [EIP](https://notes.ethereum.org/@vbuterin/verkle_tree_eip)](./verkle.png)
```

Comment:
Alt tags should describe the specific contents of an image, not simply what it is.

We're better leaving this alt blank (and probably others) as the current description doesn't describe to those what the image actually conveys.

```suggestion
![](./verkle.png)
```

(Alternatively we could create really descriptive alts, but let's just consider this a11y debt for now)